mesonbuild/interpreter/interpreter.py

```diff
@@ -2036,10 +2036,11 @@ def func_install_man(self, node: mparser.BaseNode,
     @FeatureNew('install_emptydir', '0.60.0')
     @typed_kwargs(
         'install_emptydir',
-        INSTALL_MODE_KW
+        INSTALL_MODE_KW,
+        KwargInfo('install_tag', (str, NoneType))
```

Comment:
Since this only got added in a later version, it will need to be marked `since` the version we are currently developing for.

Reply:
Which is the correct version to use?